### DISCUSSION TITLE
Star difficulties are inconsistent in ui

### DISCUSSION THREAD

**Discussion by SollyBunny** [ppy/osu#15548] [Q&A] [answered]
In the choose map ui and browse ui the difficulty of songs when you hover over their difficulty doesn't match when you actually load it. Normally it's between 0.1 and 0.5 stars below what it actually is. I don't think it's related to mods since the difference is consistent even after restarting
![unknown](https://user-images.githubusercontent.com/76408142/141027021-29e91937-c965-4dad-b9e3-327f3a05feb5.png)
![unknown](https://user-images.githubusercontent.com/76408142/141027044-f8174ba2-e115-4420-93ba-fda34d2fcfa5.png)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/discussions/15525
